packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -201,8 +201,8 @@ export interface AttachmentUploadManager {
 
 // @beta (undocumented)
 export interface AttachmentUploadOptions {
-    acceptedMediaTypes?: string[];
-    canUploadMultiple?: boolean;
+    allowedMediaTypes?: string[];
+    allowUploadMultiple?: boolean;
```

Comment:
Review had this to `disableMultipleUploads` or `disableMultipleSelection` as the default (undefined) is to allow multiple selection in the file input dialog

Reply:
I thought they wanted the prefix to consistent with allowedMediaTypes? 

I remember you said disableXXX but then Ted said to go with allowXXX or supportXXX

Reply:
updated